### PR TITLE
Change right hand side Dropdown options

### PR DESCRIPTION
**Pull request by kaankabalak**
## Description
This PR changes the order and contents of the options present in the right hand side dropdown.

## Motivation and Context
Internal discussion regarding the dropdown options

## How to test this PR?
Open right hand side dropdown and check if the URLs correspond to the links

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation needed
- [ ] Unit tests needed

### CODE REVIEW

File: browser/app/js/browser/BrowserDropdown.js [hunk 1]
```diff
@@ -88,23 +72,29 @@ export class BrowserDropdown extends React.Component {
...
-                Documentation <i className="fas fa-book" />
+              <a target="_blank" href="https://github.com/minio/minio">
+                GitHub <i className="fab fa-github" />
               </a>
             </li>
             <li>
-              <a target="_blank" href="https://slack.min.io">
-                Ask for help <i className="fas fa-question-circle" />
+              <a target="_blank" href="https://min.io/pricing?ref=ob">
+                Get Support <i className="fas fa-question-circle" />
```
Comment (harshavardhana) [inline browser/app/js/browser/BrowserDropdown.js:97]: Probably Get Support should come before GitHub @kaankabalak ?

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by kannappanr — APPROVED** [review]
LGTM

File: browser/app/js/browser/BrowserDropdown.js (hunk 1 above)
Comment (harshavardhana) [reply browser/app/js/browser/BrowserDropdown.js:97]: Probably even remove `GitHub` dropdown, I don't see why that is useful.

File: browser/app/js/browser/BrowserDropdown.js [hunk 2]
```diff
@@ -118,20 +108,9 @@ export class BrowserDropdown extends React.Component {
...
-                <ChangePasswordModal
-                  serverInfo={serverInfo}
-                  hideChangePassword={this.hideChangePassword.bind(this)}
-                />
-              )}
-            </li>
             <li>
               <a href="" id="logout" onClick={this.logout}>
-                Sign Out <i className="fas fa-sign-out-alt" />
+                Logout <i className="fas fa-sign-out-alt" />
```
Comment (harshavardhana) [inline browser/app/js/browser/BrowserDropdown.js:113]: If this is Logout, then we should have LogIn in the login screen with `alt="Login"`

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]
Discussed internally, we will improve upon this in new object browser implementation inside console

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally
Error: No such image: minio/minio:10534-f67b1c6

File: browser/app/js/browser/BrowserDropdown.js (hunk 2 above)
Comment (kaankabalak) [reply browser/app/js/browser/BrowserDropdown.js:113]: Hi @harshavardhana, from what I have understood, you want me to apply the `alt` attribute to the Login button on the main page. However, it seems like the [`alt` attribute can only be used in `img`, `area` and `input` tags](https://www.w3.org/QA/Tips/altAttribute), and the Login button only contains the `button` and `i` tags where the attribute isn't available. Where exactly should I be applying this attribute? Thanks in advance!

**Pull request merged**